storybook/stories/bar/1_basic.story.tsx

```diff
@@ -31,12 +33,69 @@ export const Example: ChartsStory = (_, { title, description }) => {
       <BarSeries
         id={specId}
         name="Simple bar series"
-        xScaleType={ScaleType.Linear}
+        xScaleType={ScaleType.Time}
         yScaleType={ScaleType.Linear}
+        enableHistogramMode
         xAccessor="x"
         yAccessors={['y']}
+        stackAccessors={['x']}
         data={data}
       />
+      <BarSeries
+        id="bars 2"
+        name="Simple bar series"
+        xScaleType={ScaleType.Time}
+        yScaleType={ScaleType.Linear}
+        enableHistogramMode
+        stackAccessors={['x']}
+        xAccessor="x"
+        yAccessors={['y']}
+        data={data}
+      />
+      <Axis id="y" position={Position.Left} domain={{ min: 0, max: 10 }} />
+      <Axis
+        id="bottom"
+        position={Position.Bottom}
+        timeAxisLayerCount={2}
+        style={{
+          tickLabel: {
+            alignment: { horizontal: 'left' },
+          },
+        }}
+      />
+      <LineAnnotation
+        id="line1"
+        domainType="xDomain"
+        style={{
+          line: {
+            opacity: 1,
+            stroke: 'red',
+            strokeWidth: 2,
+          },
+        }}
+        marker={<div style={{ width: 50, height: 50, background: 'rgba(122,40,112,0.5)' }}></div>}
```

Comment:
```suggestion
        marker={<div style={{ width: 50, height: 60, background: 'rgba(122,40,112,0.5)' }}></div>}
```

Can we make this taller?